<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>首页</string>
  </property>
  <widget class="QWidget" name="main" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>472</width>
     <height>610</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>472</width>
     <height>610</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>472</width>
     <height>610</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#main{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 214, 255, 255), stop:1 rgba(122, 217, 255, 255));
    border-radius: 20px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="topBar" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QWidget" name="userInfo" native="true">
             <property name="styleSheet">
              <string notr="true">#userInfo{
background-color: rgb(218, 255, 176);
 border-radius: 20px;
}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="avaImg">
                <property name="minimumSize">
                 <size>
                  <width>111</width>
                  <height>111</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>111</width>
                  <height>111</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">#avaImg {
	border-image: url(:/images/images/ava.jpg);
             border-radius: 55px;
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <property name="maximumSize">
                 <size>
                  <width>252</width>
                  <height>16777215</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="leftMargin">
                  <number>11</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="nickname">
                   <property name="font">
                    <font>
                     <pointsize>26</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>拯救不开心</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_7" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <item>
                       <widget class="QLabel" name="vipLabel">
                        <property name="text">
                         <string>VIP：</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="vipTime">
                        <property name="text">
                         <string>2024-12-22</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_6" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_7">
                      <item>
                       <widget class="QLabel" name="label_5">
                        <property name="text">
                         <string>直播间ID：</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="urlID">
                        <property name="enabled">
                         <bool>false</bool>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">#urlID{
border-top: none;
border-left: none;
border-right: none;
background-color:  transparent;
}</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QToolButton" name="editUrlIDBtn">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">QToolButton {
            border: none;
            background-color: transparent;
        }

QToolButton:hover{
padding-bottom:4px;
}</string>
                        </property>
                        <property name="text">
                         <string>...</string>
                        </property>
                        <property name="icon">
                         <iconset resource="pc.qrc">
                          <normaloff>:/icons/icons/create-outline.png</normaloff>:/icons/icons/create-outline.png</iconset>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item alignment="Qt::AlignRight|Qt::AlignTop">
               <widget class="QWidget" name="widget_5" native="true">
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border: none;
background-color: transparent;
}
QPushButton:hover{
padding-bottom:4px;
}</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="logOutButton">
                   <property name="minimumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="pc.qrc">
                     <normaloff>:/icons/icons/log-out-outline.png</normaloff>:/icons/icons/log-out-outline.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="settingsButton">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="pc.qrc">
                     <normaloff>:/icons/icons/settings-sharp.png</normaloff>:/icons/icons/settings-sharp.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="closeButton">
                   <property name="minimumSize">
                    <size>
                     <width>26</width>
                     <height>26</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="pc.qrc">
                     <normaloff>:/icons/icons/close-sharp.png</normaloff>:/icons/icons/close-sharp.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="menuBar" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item alignment="Qt::AlignHCenter">
            <widget class="QWidget" name="menuItem" native="true">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#menuItem{
background-color: qlineargradient(x1:0, y1:0.5, x2:1, y2:0.5, stop:0 #47a7ed, stop: 1 #a967b2);
border-radius: 20px;
}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">background-color: transparent;
image: url(:/icons/icons/logo-tiktok.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="styleSheet">
                 <string notr="true">background-color: transparent;
image: url(:/icons/icons/add.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="styleSheet">
                 <string notr="true">background-color: transparent;
image: url(:/icons/icons/midjourney.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>26</width>
                  <height>26</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="dyMjStartBtn">
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border: none;
background-color: transparent;
}
QPushButton:hover{
padding-bottom:4px;
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="pc.qrc">
                  <normaloff>:/icons/icons/power-sharp-green.png</normaloff>:/icons/icons/power-sharp-green.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="footBar" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>弹幕直播工具</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pc.qrc"/>
 </resources>
 <connections/>
</ui>
